src/routes/organization/[organizationId]/+page.server.ts

```diff
@@ -16,7 +17,8 @@ export const load = (async ({ cookies, params: { organizationId }, url }) => {
     authFunction: async ({ uid }) => await isUserOrganizationAdmin(uid, organization),
   });
   return {
-    memberDetails: JSON.stringify(await getOrganizationMemberDetails(organization)),
-    inviteDetails: JSON.stringify(await getOrganizationInviteDetails(organization)),
```

Comment:
We don't need to stringify/parse these. Idk why I did it like that originally. Benefit of not doing so is we retain typing.